qiita_pet/handlers/study_handlers/prep_template.py

```diff
@@ -77,3 +79,42 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown prep template action: %s' % action)
         self.write(result)
+
+
+class PrepFilesHandler(BaseHandler):
+    @authenticated
+    def get(self):
+        study_id = self.get_argument('study_id')
+        prep_file = self.get_argument('prep_file')
+        # atype = self.get_argument('type')
+
+        # TODO: Get file types for the artifact type
+        # FILE TYPE IN POSTION 0 MUST BE DEFAULT FOR SELECTED
+        file_types = ['raw_fwd', 'raw_rev']
+
+        selected = []
+        not_selected = []
+        _, base = get_mountpoint("uploads")[0]
+        uploaded = get_files_from_uploads_folders(study_id)
+        prep = pd.read_table(join(base, study_id, prep_file), sep='\t')
+        if 'run_prefix' in prep.columns:
```

Comment:
Why? The functionality is the same, it does work, but the problem is that the name of the variable is misleading.

run_prefix is not useful only when there are per sample files, is also useful when in a single prep template there are more than one lane. In such case, the "run_prefix" column helps to differentiate which set of samples belong to the given lane.

That is way I'm suggesting that change here, because as the code currently is makes the illusion that is only used when per_sample files are used which is not correct.